doc/classes/ProjectSettings.xml

```diff
@@ -917,6 +917,7 @@
 		</member>
 		<member name="editor/export/convert_text_resources_to_binary" type="bool" setter="" getter="" default="true">
 			If [code]true[/code], text resources are converted to a binary format on export. This decreases file sizes and speeds up loading slightly.
+			[b]Note:[/b] Project Settings are not a resource so "project.godot" will always be converted to "project.binary" on export.
```

Comment:
```suggestion
			[b]Note:[/b] [ProjectSettings] is not a resource so "project.godot" will always be converted to "project.binary" on export.
```

Reply:
I'd also make it `[code]project.godot[/code]` and `[code]project.binary[/code]`

Reply:
Thanks!